backend/.docker/app-prod/php-fpm.sh

```diff
@@ -8,6 +8,12 @@
 # file that was distributed with this source code.
 
 console cache:warmup --no-interaction
+
+if [ "${APP_RESET_ON_BUILD:-}" = true ]; then
```

Comment:
```suggestion
if [ "${RESET_DATABASE:-}" = true ]; then
```

This change is suggested because the app is not reset, the database is. And because it is not reset on build, it is reset every time this image is run.